interface/forms/sdoh/new.php

```diff
@@ -0,0 +1,1129 @@
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="care5to12" name='care5to12' <?php echo (($obj["care5to12"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="care5to12"><?php echo xlt('Children age 5 to 12');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="careover12" name='careover12' <?php echo (($obj["careover12"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="careover12"><?php echo xlt('Children over 12');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="carespecneeds" name='carespecneeds'  <?php echo (($obj["carespecneeds"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="carespecneeds"><?php echo xlt('Special Needs Child');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="caredisabled" name='caredisabled' <?php echo (($obj["caredisabled"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="caredisabled"><?php echo xlt('Disabled or Ill Adult');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="careelderly" name='careelderly' <?php echo (($obj["careelderly"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="careelderly"><?php echo xlt('Elderly');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="careother" name='careother' <?php echo (($obj["careother"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="careother"><?php echo xlt('Other');?></label>
+                                                <input type="text" id="careotherinput" name='careotherinput' size="30" value="<?php echo text($obj["careotherinput"] ?? ''); ?>"/>
```

Comment:
for these cases (the other input) will use attr() instead of text() (we use attr() when escaping html attributes, such as this case where escaping the html value attribute)